src/utils/convertExperimentToApiModel.js

```diff
@@ -0,0 +1,20 @@
+// There are some differences between the property naming of elements
+// stored in the ui and in the api,
+// this is an attempt to deal with this in one single place (the ui)
+// We should try to converge to one single model to follow
+const experimentConvertedToApiModel = (experiment) => {
```

Comment:
can you put this new file into a new folder inside `src/utils`? I imagine that there will be functions for converting other data too, it will be good to give it some structure.

Reply:
based on [this comment](https://github.com/hms-dbmi-cellenics/ui/pull/652#discussion_r823550791), I think we can skip moving this into a folder sincre we are going to get rid of this function soon anyways.